README.md

```diff
@@ -262,10 +257,13 @@ that you wish the world to use:
 
 ### Adjusting global server properties
 Default values in the script can be overridden by adding certain properties to one
-of the `mscs.defaults` files. The `mscs.defaults` files can be found found in one
+of the `mscs.defaults` files. The `mscs.defaults` files needs to be created in one
```

Comment:
Automatically generating a file for this location might not be a bad idea.  Either commented out (assuming the code can handle that, I think it does), or simply with the default values.